tests/conftest.py

```diff
@@ -17,6 +17,11 @@ def pytest_addoption(parser):
         'This GCS bucket must be publicly accessible '
         '(read access for everyone is enough for testing).',
     )
+    parser.addoption(
+        '--gcp-res-analysis-metadata',
+        default='gs://caper-data/gcp_resource_analysis/out/atac/e5eab444-cb6c-414a-a090-2c12417be542/metadata.json',
+        help='GCS path of metadata.json file to test gcp_res_analysis.',
+    )
```

Comment:
Seems kind of easy to forget to use this option when running tests. Why not just use a fixture?

Reply:
It was already a fixture depending on argparse. I de-parameterized it as you suggested.